lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -15,14 +15,14 @@ defmodule Kernel.LexicalTracker do
   Returns all remotes linked to in this lexical scope.
   """
   def remotes(arg) do
-    :gen_server.call(to_pid(arg), :ets, @timeout)
-    |> :ets.match({{:mode, :'$1'}, :'$2'})
+    references = :gen_server.call(to_pid(arg), :references, @timeout)
+    for({{:mode, module}, mode} <- references, do: {module, mode})
```

Comment:
I think we should likely filter `references` inside `gen_server` process to avoid copying.